### ISSUE TITLE
TFS Windows Notifications only when I triggered a build

### ISSUE THREAD

**Issue by schulzi66** [AnyStatus/Support#74]
I would love, if I can configure for my TFS Builds to only get a windows notification for my triggered builds. Right now I get a ton of notifications for all build, even if another developer triggered it. 

Is there any plan for this requirement?

**Comment by AlonAm** [comment]
That's a good idea. I suppose it is doable but it may take some time to implement as there are requirements in higher priority such as TFS releases.
Feel free to contribute that change yourself, I'd love to assist you with that.
Thanks for the feedback :-)

**Comment by schulzi66** [comment]
Yes I looked at the code and I think it is doable. I hope to find some time at the weekend to implement it. 
Keep up the work! I think this extension can become very valuable for me 👍 
If I need help, I will let you know :)

**Comment by schulzi66** [comment]
Should it be implemented in VSTSBuild or in TFSBuild? Because right now have configured my anystatus in visual studio to use tfs builds

**Comment by AlonAm** [comment]
If you're using TFS then TFSBuild
